server/src/lib/storage/data/92c9f930-bc91-11e8-9a34-994221d21d91.json

```diff
@@ -0,0 +1 @@
+{"id":"92c9f930-bc91-11e8-9a34-994221d21d91","lastUpdated":"2018-09-20T04:56:45.381Z","createdOn":"2018-09-20T04:56:45.381Z","title":"noted","content":"contented"}
```

Comment:
You wouldn't want to leave these test files inside your GH project since these are for testing on your local environment.